components/lib/publicsuffixlist/src/main/java/mozilla/components/lib/publicsuffixlist/PublicSuffixList.kt

```diff
@@ -59,7 +59,7 @@ class PublicSuffixList(
     }
 
     /**
-     * Returns the public suffix and one more level; known as the effective top-level domain (eTLD). Returns `null` if
+     * Returns the public suffix and one more level; known as the registrable domain domain. Returns `null` if
```

Comment:
I guess this is mistake domain domain?

Reply:
Oh yes, I fixed that locally but apparently I didn't push :D

Reply:
👍

Reply:
Pushed :)